src/optimizer/simple_optimizer.cpp

```diff
@@ -103,7 +104,8 @@ std::shared_ptr<planner::AbstractPlan> SimpleOptimizer::BuildPelotonPlanTree(
         try {
           catalog::Catalog::GetInstance()->GetTableWithName(DEFAULT_DB_NAME,
                                                             "order_line");
-          auto child_SelectPlan = CreateHackingJoinPlan();
+          // auto child_SelectPlan = CreateHackingJoinPlan();
```

Comment:
If CreateHackingJoinPlan() is no longer used anymore, we should just delete it. You don't have to do this now, but make sure you clean it up after we finish all the stuff for NestedLoopJoin.